### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by TudorVersoiu** [EHfive/pulseaudio-modules-bt#151]
Aptx HD not available on Sennheiser HD 350bt

Bluemann fails to change the audio profile to Aptx HD. Pavucontrol shows it as unavailable.

Distribution: Manjaro Manjaro 20.2.1 Nibia(kernel 4.x)
Pulseaudio 14.0
Both AUR git version and precompiled packages tested with the same result.

Output of `avinfo`:
```
Stream End-Point #8: Audio Sink 
	Media Codec: Vendor Specific A2DP Codec
		Vendor ID 0x000000d7
		Vendor Specific Codec ID 0x0002
		Vendor Specific Data: 0x32 0x02 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00
	Content Protection: SCMS-T
Stream End-Point #6: Audio Sink 
	Media Codec: Vendor Specific A2DP Codec
		Vendor ID 0x0000000a
		Vendor Specific Codec ID 0x0002
		Vendor Specific Data: 0x32 0x02 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00
		Vendor Specific Value (aptX Low Latency)
			Frequencies: 44.1kHz 48kHz 
			Channel modes: Stereo 
		Bidirectional link: No
		Target codec buffer level: 0
		Initial codec buffer level: 0
		SRA max rate: 0
		SRA averaging time: 0s
		Good working codec buffer level: 0
	Content Protection: SCMS-T
Stream End-Point #5: Audio Sink 
	Media Codec: Vendor Specific A2DP Codec
		Vendor ID 0x0000004f
		Vendor Specific Codec ID 0x0001
		Vendor Specific Data: 0x32
		Vendor Specific Value (aptX)
			Frequencies: 44.1kHz 48kHz 
			Channel modes: Stereo 
	Content Protection: SCMS-T
Stream End-Point #3: Audio Sink 
	Media Codec: MPEG24
		Object Types: MPEG-2 AAC LC 
		Frequencies: 8kHz 11.025kHz 12kHz 16kHz 22.05kHz 24kHz 32kHz 44.1kHz 48kHz 
		Channels: 1 2 
		Bitrate: 320000
		VBR: Yes
	Content Protection: SCMS-T
Stream End-Point #1: Audio Sink 
	Media Codec: SBC
		Channel Modes: Mono DualChannel Stereo JointStereo
		Frequencies: 16Khz 32Khz 44.1Khz 48Khz 
		Subbands: 4 8
		Blocks: 4 8 12 16 
		Bitpool Range: 2-53
	Content Protection: SCMS-T
```

I think that the vendor ID's for Aptx HD that sennheiser uses are non standard.(I've seen something similar on another issue)
```
Stream End-Point #8: Audio Sink 
	Media Codec: Vendor Specific A2DP Codec
		Vendor ID 0x000000d7
		Vendor Specific Codec ID 0x0002
```
I think that here "Vendor Specific Codec ID should have been 0x0024 and that's why it's not being recognized.

**Comment by TudorVersoiu** [comment]
Nevermind, after a double check I found out that contrary to what I've read somewhere about them, they don't actually support Aptx HD, just Aptx, so this should probably be deleted.
